aria/storage/models.py

```diff
@@ -251,13 +278,21 @@ class Node(Model):
     max_number_of_instances = Field(type=int)
 
     def relationships_by_target(self, target_id):
+        """
+        Retreives all of the relationship by target.
+        :param target_id: the node id of the target  of the relationship
+        :return:
```

Comment:
return statement's empty; since you're yielding it probably requires a different syntax.
i know google's syntax has a ":yields:" keyword, but since we haven't settled on a format yet i dont know if it should be used or simply have the return removed..